policies/ip-policy.md

```diff
@@ -33,21 +30,19 @@ The Project Documentation License can be any OSI-approved license or a Creative
 
 ## Contributions
 
-> [!TIP]
-> All code contributions must align with the project’s license and come with a promise (DCO) from contributors that they have the right to contribute under these terms.
-
-All new code contributions to any Project shall be made under the Project Code License accompanied by a Developers Certificate of Origin (DCO, available at <http://developercertificate.org/>), which will bind the individual contributor and, if applicable, their employer to the Project Code License.
+All new code contributions to any Project must be made under the Project Code License. Contributions must be accompanied by either a Developer Certificate of Origin (DCO available at <http://developercertificate.org/>) or a Contributor License Agreement (CLA), as determined by the project’s technical governing body. The DCO or CLA binds the individual contributor and, if applicable, their employer to the Project Code License.
```

Comment:
I'm assuming it is implied that all projects have either a dco or cla in effect for any new contributors?

If it is - should it not be explicitly stated as reading this a project don't need one - just that if it has either dco or cla those are in effect.

Reply:
I am not sure what is unclear.

All new code contributions to a CF project must follow the project code license and be accompanied by either the DCO or a CLA (as chosen by the project). I don't think there is anything in that text that says "neither" or "none" is a valid option.